docs/batch_size_optimizer/server.md

```diff
@@ -0,0 +1,156 @@
+# Batch Size Optimizer in Zeus
+
+## What is it?
+
+Batch size optimizer(BSO) can choose the best batch size that minimizes the cost, where cost is defined as $cost = \eta \times \text{Energy consumption to accuracy} + (1-\eta) \times \text{Max power}\times \text{Time to accuracy}$.
+
+## How does it work?
+
+Core of BSO is a multi-arm-bandit based on **recurrent** trainings. After each training, we feed the result cost to MAB and after certain number of trainings, MAB can converge to the best batch size. In addition to MAB, we employed early-stopping and pruning to handle stragglers. For more detail, refer to [paper](https://www.usenix.org/conference/nsdi23/presentation/you).
+
+## Should I use this?
+
+The key of BSO is recurrent training. If you are training your model periodically or repeatedly, BSO can be a great choice to reduce energy or time consumption.
+
+## Limitations
+
+We currently doesn't support heterogeneous GPUs or different configurations. Number of GPUs, gpu models, and other configurations in JobSpec should be identical in recurrent trainings. If you are running your training in a various environment each time, then it might not desirable to use BSO.
+
+## Quick start (Server)
+
+1. Clone the repository
+
+    ```Shell
+    git clone https://github.com/ml-energy/zeus/tree/master
+    ```
+
+2. Create `.env` under `/docker`. Example of `.env` is provided below.
+
+    For default, we are using the MySQL for the database.
+
+    ```Shell
```

Comment:
What are other configuration options available?

Reply:
https://github.com/jincheng9/markdown_supported_languages?tab=readme-ov-file#heres-a-full-list-of-supported-languages

Reply:
?

Reply:
Oh lol got it. I wasn't referring to markdown. I mean other env vars.

Reply:
oh that is the complete list of env vars.

Reply:
Haha gotcha, just wanted to have a complete list of configs.